server/checks/Approval.js

```diff
@@ -15,15 +17,26 @@ export default class Approval extends Check {
   static HOOK_EVENTS = [EVENTS.PULL_REQUEST, EVENTS.ISSUE_COMMENT]
 
   /**
-   * Based on the Zappr approval configuration and the approval statistics,
-   * generates a commit status object that it consumed by the Github Status API
+   * Based on the Zappr approval configuration
+   * and the approval statistics and number of vetos.
+   * Generates a commit status object that it consumed by the Github Status API
    * (https://developer.github.com/v3/repos/statuses/#create-a-status).
    *
    * @param approvals Approval stats. Includes `total` approvals and group information.
+   * @param vetos Number of vetos.
    * @param approvalConfig Approval configuration
    * @returns {Object} Object consumable by Github Status API
    */
-  static generateStatus(approvals, {minimum, groups}) {
+  static generateStatus(approvals, vetos, {minimum, groups}) {
```

Comment:
I'd prefer to have approvals and vetos combined in one object that contains everything we found in the issue comments.